feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt

```diff
@@ -110,170 +115,297 @@ private fun HomeScreen(
             item {
-                Text(
-                    modifier = Modifier.padding(top = 32.dp),
-                    text = "Recent Transactions",
-                    style = MaterialTheme.typography.bodyMedium,
-                    color = MaterialTheme.colorScheme.onSurface,
+                TransactionHistoryCard(
+                    modifier = Modifier.padding(vertical = 20.dp),
+                    transactions = transactions,
                 )
-                Spacer(modifier = Modifier.height(16.dp))
             }
         }
-        items(transactions) { transaction ->
-            TransactionItemScreen(transaction = transaction)
-        }
-        item {
-            Spacer(modifier = Modifier.height(24.dp))
-        }
     }
 }
 
 @Composable
-private fun MifosWalletCardScreen(
+private fun MifosWalletCard(
     modifier: Modifier = Modifier,
     account: Account? = null,
 ) {
-    Card(
+    Box(
         modifier = modifier
             .fillMaxWidth()
-            .height(225.dp)
-            .padding(top = 20.dp, bottom = 32.dp),
-        colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.onSurface),
+            .height(200.dp)
+            .background(
+                brush = Brush.linearGradient(
+                    colors = listOf(
+                        NewUi.walletColor1,
+                        NewUi.walletColor2,
+                    ),
+                ),
+                shape = RoundedCornerShape(16.dp),
+            ),
     ) {
-        Column(
+        Card(
             modifier = Modifier
-                .fillMaxHeight()
-                .padding(start = 36.dp),
-            verticalArrangement = Arrangement.Center,
+                .fillMaxSize(),
+            colors = CardDefaults.cardColors(
+                containerColor = Color.Transparent,
+            ),
         ) {
-            val walletBalanceLabel =
-                if (account != null) "(${account.currency.displayLabel})" else ""
-            Text(
-                text = "Wallet Balance $walletBalanceLabel",
-                style = TextStyle(
-                    fontSize = 12.sp,
-                    fontWeight = FontWeight.W400,
-                    color = lightGrey,
-                ),
-            )
-            Spacer(modifier = Modifier.height(10.dp))
-            val accountBalance =
-                if (account != null) {
-                    Utils.getFormattedAccountBalance(
-                        account.balance,
-                        account.currency.code,
+            Column(
+                modifier = Modifier.fillMaxSize(),
+            ) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.SpaceBetween,
+                ) {
+                    Column(modifier = Modifier.padding(20.dp)) {
+                        Text(
+                            text = "Client Name",
```

Comment:
Show the Client Name @itsPronay

Reply:
Don't hardcode the text, show client name instead

Reply:
![image](https://github.com/user-attachments/assets/a18b8bab-1848-45c0-90ee-63bba0bac0de)
we are already showing here.

Reply:
in the demo API, account?.name = 'WALLET'

Reply:
Oh! Okay